pkg/hostpath/nodeserver.go

```diff
@@ -189,6 +244,7 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 			err.Error()))
 	}
 
+	storeVolMapToDisk()
```

Comment:
Check for error and raise here.

Reply:
update for this requested change pushed @adambkaplan